CRM/Contact/Import/Parser/Contact.php

```diff
@@ -550,7 +546,7 @@ public function import($onDuplicate, &$values) {
       CRM_Utils_Hook::import('Contact', 'process', $this, $hookParams);
     }
 
-    if ($relationship) {
+    if (1) {
```

Comment:
@eileenmcnaughton that means the else part is unreachable right?

Reply:
@monishdeb yes - I hadn't picked that up - but if you merge this I will remove the IF altogether & the else & do whitespace in a follow up

Reply:
cool 👍